diff -Naur A/Makefile.in B/Makefile.in
--- A/Makefile.in	2020-09-28 13:13:49.470992040 +0200
+++ B/Makefile.in	2020-09-28 13:14:22.871152176 +0200
@@ -4388,7 +4388,7 @@
 dist_libexec_SCRIPTS = $(am__append_33)
 
 ###############################################################################
-lib_LTLIBRARIES = libnm/libnm.la $(am__append_69)
+lib_LTLIBRARIES = $(am__append_69)
 plugin_LTLIBRARIES = $(core_plugins)
 
 ###############################################################################