diff -Nuar libbsd-0.8.3.orig/include/bsd/sys/cdefs.h libbsd-0.8.3/include/bsd/sys/cdefs.h
--- libbsd-0.8.3.orig/include/bsd/sys/cdefs.h	2015-09-23 05:59:27.000000000 +0000
+++ libbsd-0.8.3/include/bsd/sys/cdefs.h	2017-04-24 13:11:51.476734210 +0000
@@ -24,11 +24,13 @@
  * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+/*
 #ifdef LIBBSD_OVERLAY
 #include_next <sys/cdefs.h>
 #else
 #include <sys/cdefs.h>
 #endif
+*/
 
 #ifndef LIBBSD_SYS_CDEFS_H
 #define LIBBSD_SYS_CDEFS_H
diff -Nuar libbsd-0.8.3.orig/src/flopen.c libbsd-0.8.3/src/flopen.c
--- libbsd-0.8.3.orig/src/flopen.c	2016-03-27 11:33:21.000000000 +0000
+++ libbsd-0.8.3/src/flopen.c	2017-04-24 13:11:51.477734208 +0000
@@ -36,6 +36,7 @@
 #include <stdarg.h>
 #include <unistd.h>
 #include <libutil.h>
+#include <fcntl.h>
 
 int
 flopen(const char *path, int flags, ...)
diff -Nuar libbsd-0.8.3.orig/src/nlist.c libbsd-0.8.3/src/nlist.c
--- libbsd-0.8.3.orig/src/nlist.c	2016-03-27 11:33:21.000000000 +0000
+++ libbsd-0.8.3/src/nlist.c	2017-04-24 13:12:20.099661591 +0000
@@ -40,10 +40,11 @@
 
 #include <errno.h>
 #include <fcntl.h>
-#include <a.out.h>
+#include <linux/a.out.h>
 #include <stdio.h>
 #include <string.h>
 #include <unistd.h>
+#include <fcntl.h>
 
 #if !defined(__NO_A_OUT_SUPPORT)
 #define _NLIST_DO_AOUT
