diff -Nuar serf-9999.orig/SConstruct serf-9999/SConstruct
--- serf-9999.orig/SConstruct	2018-04-12 19:50:22.862712711 +0200
+++ serf-9999/SConstruct	2018-04-12 19:51:52.231707768 +0200
@@ -115,6 +115,9 @@
   BoolVariable('DISABLE_LOGGING',
                "Disable the logging framework at compile time",
                False),
+  RawListVariable('AR', "Command name or path of the archiver", None),
+  RawListVariable('RANLIB', "Command name or path of the archiver indexer", None),
+  BoolVariable('BUILD_STATIC', 'Build libserf static library', True),
   RawListVariable('CC', "Command name or path of the C compiler", None),
   RawListVariable('CFLAGS', "Extra flags for the C compiler (space-separated)",
                   None),
@@ -215,6 +218,7 @@
 
 debug = env.get('DEBUG', None)
 aprstatic = env.get('APR_STATIC', None)
+build_static = env.get('BUILD_STATIC', True)
 disablelogging = env.get('DISABLE_LOGGING', None)
 
 Help(opts.GenerateHelpText(env))
@@ -515,7 +519,9 @@
                                                           brotli_libs),
                            })
 
-env.Default(lib_static, lib_shared, pkgconfig)
+env.Default(lib_shared, pkgconfig)
+if build_static:
+  env.Default(lib_static)
 
 if CALLOUT_OKAY:
   conf = Configure(env)
@@ -547,8 +553,10 @@
                                      % (target_install_shared_path,
                                         install_shared_path)))
 
-env.Alias('install-lib', [install_static, install_shared,
-                          ])
+install_libs = [install_shared]
+if build_static:
+  install_libs.append(install_static)
+env.Alias('install-lib', install_libs)
 env.Alias('install-inc', env.Install(incdir, HEADER_FILES))
 env.Alias('install-pc', env.Install(os.path.join(libdir, 'pkgconfig'),
                                     pkgconfig))
