diff -Nuar qtwebengine-everywhere-src-5.12.1.orig/src/3rdparty/chromium/third_party/blink/renderer/platform/wtf/stack_util.cc qtwebengine-everywhere-src-5.12.1/src/3rdparty/chromium/third_party/blink/renderer/platform/wtf/stack_util.cc
--- qtwebengine-everywhere-src-5.12.1.orig/src/3rdparty/chromium/third_party/blink/renderer/platform/wtf/stack_util.cc	2019-01-16 11:59:47.000000000 +0100
+++ qtwebengine-everywhere-src-5.12.1/src/3rdparty/chromium/third_party/blink/renderer/platform/wtf/stack_util.cc	2019-03-16 14:28:59.516994405 +0100
@@ -28,7 +28,7 @@
 // FIXME: On Mac OSX and Linux, this method cannot estimate stack size
 // correctly for the main thread.
 
-#if defined(__GLIBC__) || defined(OS_ANDROID) || defined(OS_FREEBSD) || \
+#if defined(OS_LINUX) || defined(OS_ANDROID) || defined(OS_FREEBSD) || \
     defined(OS_FUCHSIA)
   // pthread_getattr_np() can fail if the thread is not invoked by
   // pthread_create() (e.g., the main thread of webkit_unit_tests).
@@ -96,7 +96,7 @@
 }
 
 void* GetStackStart() {
-#if defined(__GLIBC__) || defined(OS_ANDROID) || defined(OS_FREEBSD) || \
+#if defined(OS_LINUX) || defined(OS_ANDROID) || defined(OS_FREEBSD) || \
     defined(OS_FUCHSIA)
   pthread_attr_t attr;
   int error;
