diff -Nuar zenity-3.44.1.orig/meson.build zenity-3.44.1/meson.build
--- zenity-3.44.1.orig/meson.build	2023-04-28 15:59:40.000000000 +0200
+++ zenity-3.44.1/meson.build	2023-08-04 09:17:03.452650693 +0200
@@ -81,6 +81,13 @@
 
 # link Xlib if we have it. This will likely be removed after gtk4 migration.
 x11 = dependency('x11', required: false)
+if get_option('x11')
+  if x11.found()
+    zenity_conf.set('GDK_WINDOWING_X11', true)
+  else
+    error(opt_missing_str.format('x11'))
+  endif
+endif
 
 perl = find_program('perl', required: false)
 if perl.found()
@@ -102,6 +109,7 @@
 
 summary({'libnotify': get_option('libnotify'),
          'webkitgtk': get_option('webkitgtk'),
+         'x11': get_option('x11'),
          'gdialog script': perl.found(),
         }, section: 'Configuration')
 
diff -Nuar zenity-3.44.1.orig/meson_options.txt zenity-3.44.1/meson_options.txt
--- zenity-3.44.1.orig/meson_options.txt	2023-04-28 15:59:40.000000000 +0200
+++ zenity-3.44.1/meson_options.txt	2023-08-04 09:14:10.622795209 +0200
@@ -3,3 +3,6 @@
 
 option('webkitgtk', type : 'boolean', value : false,
   description : 'enable webkitgtk support')
+
+option('x11', type : 'boolean', value : false,
+  description : 'enable X11 support')
